packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
```

Comment:
The docstrings on the new attributes should be complete English sentences, and, ideally add more information beyond the variable's name. For example:

```dart
/// The background color of the header when the stepper is in [StepperType.horizontal] mode.
///
/// The default color is `primaryColor` from the current theme.
final Color? backgroundHeaderColor;

```